package.json

```diff
@@ -8,6 +8,7 @@
   "license": "GPL-3.0",
   "dependencies": {
     "irc": "^0.5.2",
+    "jsonfile": "^3.0.0",
```

Comment:
@ryzokuken Its same 2 lines of code. You don't need to put it in utils. See https://stackoverflow.com/a/10011078